content/docs/design/visual_story.md

```diff
@@ -19,8 +19,7 @@ In this tutorial, we'll introduce you to the [amp-story](/docs/reference/compone
 - Animate elements on a page
 - Keep readers engaged with your content by adding related links to the end of the story
 
-{% call callout('Note', type='note') %} The [amp-story](/docs/reference/components/amp-story.html) component is **experimental**. To sign up for the origin trial to publish pages with this component, please visit <a href="http://bit.ly/amp-story-signup">bit.ly/amp-story-signup</a>. {% endcall %}
-
+{% call callout('Note', type='note') %} The [amp-story](/docs/reference/components/amp-story.html) component is **experimental**. But, it's [open to all developers without origin trials or a whitelist](/latest/blog/new-in-amp-stories-monetization-revamped-bookends-and-metadata/). {% endcall %}
```

Comment:
I'd recommend dropping any mention of origin or whitelist, simply keep the first sentence.

Reply:
done.